c2corg_ui/static/js/apiservice.js

```diff
@@ -350,4 +359,38 @@ app.Api.prototype.updateAccount = function(data) {
 };
 
 
+
+/**
+ * @param {Object} file
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
+ */
+app.Api.prototype.uploadImage = function (file) {
+  var defer = this.q_.defer();
+  var loaded = 0;
+  setInterval(function () {
+    file['progress']++;
+    defer.notify({'loaded': (file['progress']/100) * file['size'], 'total': file['size']}); // for the progress function
+    if (file['progress'] >= 100) {
+      file['metadata']['server_file_name'] = '23259810.jpg';
+      defer.resolve(file);
+    }
+  }, 50);
+  return defer.promise;
+};
+
+
+/**
+ * Saves the images and sends the metadatas (exif, title, server_file_name, activities...)
+ * @param {Array<File>} files
+ */
+app.Api.prototype.saveImages = function(files) {
```

Comment:
Should we have a dedicated endpoint in API or use the document creation endpoint with new attributes?